Makefile

```diff
@@ -24,7 +24,54 @@ overfit:
 smalldata:
 	$(PYTEST) tests -v -m "smalldata"
 
-SNLI:
-	$(PYTEST) tests -v -m SNLI
+
+# ===========
+# SNLI
+# ===========
+
+snli:
+	./data/SNLI/download.sh
+	$(PYTHON) jack/io/SNLI2jtr.py
+snli-esim:
+	 $(PYTHON) bin/jack-train.py with \
```

Comment:
we should never use the standard jack.yaml config, but inherit from it and create new configs for each model or at least task. E.g. you could create a `conf/natural_language_inference.yaml` which by default trains a model on snli. See the `conf/fastqa.yaml` for instance. in the command line we only need to overwrite the model type (e.g., esim, bilstm, ...)